App.tsx

```diff
@@ -1,19 +1,43 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import OnboardingScreen from './screens/Onboarding/OnboardingScreen';
+//import InformationScreen from './screens/Registration/InformationScreen/InformationScreen'
+//import InterestsScreen from './screens/Registration/InterestsScreen/InterestsScreen'
+
+import * as Font from 'expo-font';
+import React, { useState, useEffect,  } from 'react';
+
+
+
+
 
 export default function App() {
+  const[fontsLoaded, setFontsLoaded] = useState(false);
+
+  //this loads the fonts to be used in the app
+  useEffect(() => {
+    const loadFonts = async () => {
+    await Font.loadAsync({
+      'JetBrains-Mono': require('./assets/fonts/JetBrainsMono-Regular.ttf'),
+
+    })
+    setFontsLoaded(true)
+  }
+  loadFonts()
+  }, [])
+
+  if(!fontsLoaded){
+    //loading screen
+    return;
+  }
+
```

Comment:
Add a `useEffect` hook to hide the splash screen when the fonts are loaded.

```suggestion
 useEffect(() => {
    if (fontsLoaded) {
      SplashScreen.hideAsync()
    }
  }, [fontsLoaded])
```